scripts/linux/runE2ETest.sh

```diff
@@ -169,12 +169,10 @@ function prepare_test_from_artifacts() {
 
                 local escapedSnitchAlertUrl
                 local escapedBuildId
-                local tracking_id=$(cat /proc/sys/kernel/random/uuid)
```

Comment:
why move this? tracking id should only be used for long haul and stress test, right? that's why this line is put in this section.  I think adding tracking id to TESTINFO should be put here as well.

Reply:
Same in Run-E2ETest.ps1

Reply:
This is moved because we need to add this field to TestInfo. I think the cleanest place to do that is with all the default value setup near EOF after process arguments. Let me know if you disagree.

Reply:
my point is, we should not run any line of code if it is not needed.  Tracking id is only used for long haul and stress test, we should not run that line if it is running other e2e test.  I can see 2 options; 1) keep tracking id assignment at EOF but add if-statement to run only for stress and long haul; 2) move tracking id and testinfo assigment back to here. Or you have another idea.

Reply:
Oh this is rly good point. I think I prefer 1)